opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/authenticate/embedded/Xs2aAskForIban.java

```diff
@@ -0,0 +1,71 @@
+package de.adorsys.opba.protocol.xs2a.service.xs2a.consent.authenticate.embedded;
+
+import de.adorsys.opba.protocol.xs2a.service.ContextUtil;
+import de.adorsys.opba.protocol.xs2a.service.ValidatedExecution;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.RedirectExecutor;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.context.Xs2aContext;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.context.ais.Xs2aAisContext;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.consent.AisConsentInitiateBody;
+import lombok.RequiredArgsConstructor;
+import org.flowable.engine.RuntimeService;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Service("xs2aAskForIban")
+@RequiredArgsConstructor
+public class Xs2aAskForIban extends ValidatedExecution<Xs2aContext> {
+
+    private final RuntimeService runtimeService;
+    private final RedirectExecutor redirectExecutor;
+
+    @Override
+    protected void doRealExecution(DelegateExecution execution, Xs2aContext context) {
+        redirectExecutor.redirect(execution, context, redir -> redir.getParameters().getProvidePsuIban());
+    }
+
+    @Override
+    protected void doMockedExecution(DelegateExecution execution, Xs2aContext context) {
```

Comment:
There should be no overrides here - mocked execution should be no-op

Reply:
done